.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2889,7 +2889,15 @@ stages:
             az storage blob upload --container-name "$(AZURE_STORAGE_CONTAINER_NAME)" --file "sha.txt" --name "sha.txt" --overwrite true
             az storage blob upload --container-name "$(AZURE_STORAGE_CONTAINER_NAME)" --file "version.txt" --name "version.txt" --overwrite true
           displayName: Upload indexes to Azure
-          condition: and(succeeded(), eq(variables.isMainBranch, true))
+          condition: >
```

Comment:
This is the only actual change, the rest is just whitespace clean up from VS code. 

I left the whitespace cleanup, as I thought it was probably useful.